Gruntfile.js

```diff
@@ -63,16 +83,28 @@ module.exports = function(grunt) {
         watch: {
             js: {
                 files: ['js/**/*.js'],
-                tasks: ['concurrent', 'concat', 'uglify']
+                tasks: ['jslint', 'concat', 'uglify']
+            },
+            css: {
+                files: ['css/**/*.scss'],
+                tasks: ['scsslint', 'sass']
             }
         },
 
         concurrent: {
-            lint: ['jshint', 'jscs']
+            jslint: ['jslint'],
+            js: [['concurrent:jslint', 'concat', 'uglify']],
```

Comment:
That makes sense, I'll get this updated when I get reliable internet.

As a sidenote, calling the `jslint` task (that is defined separately) doesn't make sense does it? By doing this, the tasks inside it (`jshint` and `jscs`) are no longer running concurrently?